pkgs/applications/office/ledger/default.nix

```diff
@@ -1,23 +1,24 @@
 { stdenv, lib, fetchFromGitHub, cmake, boost, gmp, mpfr, libedit, python3
-, fetchpatch, installShellFiles, texinfo, gnused, usePython ? true }:
+, installShellFiles, texinfo, gnused, usePython ? false }:
```

Comment:
Given that ledger 3.3.0 adds support for encrypted files using GnuPG, it would be helpful to have a useGpgme option similar to usePython